packages/cli/src/commands/build/index.ts

```diff
@@ -820,60 +813,51 @@ function mergeWildcard(
   return wildcard;
 }
 
-function mergeFlags(
-  buildResults: Iterable<BuildResult | BuildOutputConfig>
-): BuildResultV2Typical['flags'] {
-  return Array.from(buildResults).flatMap(result => {
-    if ('flags' in result) {
-      return result.flags ?? [];
-    }
-
-    return [];
-  });
-}
-
 /**
- * Takes the build output and writes all the variants into the `variants.json`
- * file. It'll skip variants that already exist.
+ * Takes the build output and writes all the flags into the `flags.json`
+ * file. It'll skip flags that already exist.
```

Comment:
What does the "It'll skip flags that already exist." part refer to?  Trying to understand this

Reply:
@dferber90 There is a chance that multiple builds, e.g. `@vercel/next` and `@vercel/node` being run on the same project, emit flags with the same name.

This is very unlikely, since it's more or less expected that only the main build, and eventually only the Build Output API has flags, but this is mostly to satisfy legacy behaviour and types.